config/initializers/find_each_with_order.rb

```diff
@@ -0,0 +1,18 @@
+Rails.logger.debug "Running initializer in #{__FILE__}"
+# From https://veerasundaravel.wordpress.com/2014/11/26/rails-find_each-method-with-order-option/
+class ActiveRecord::Relation
+  # normal find_each does not use given order but uses id asc
+  def find_each_with_order(batch_size: 1_000)
```

Comment:
I think we should remove this since you aren't using it. It looks safe enough, but stale monkey patches scare me.